### ISSUE TITLE
Can I found my report when  "Cannot chdir to"

### ISSUE THREAD

**Issue by GoogleCodeExporter** [tinselcity/skipfish#141]
```
Something wrong is happen when skipfish will end, and it output

[-]  SYSTEM ERROR : Cannot chdir to '/Users/username/log'
    Stop location : write_report(), report.c:824
       OS message : No such file or directory


Can I found my report when this is happen?
```

Original issue reported on code.google.com by `lne1...@gmail.com` on 14 Feb 2012 at 8:17

**Comment by GoogleCodeExporter** [comment]
```
Can you give me the command-line and output of "ls -ld /Users/username/log" 

Does this happen everytime?  Note that the output dir gets created during scan 
startup so if it gets removed during the course of the scan, you will get this 
error.
```

Original comment by `niels.he...@gmail.com` on 20 Feb 2012 at 3:00

**Comment by GoogleCodeExporter** [comment]
```
This error  happen only once.

Maybe I delete the log dir when skipfish is running...

Is there a temporary floder to save the report when this error happen?
```

Original comment by `lne1...@gmail.com` on 21 Feb 2012 at 1:20

**Comment by GoogleCodeExporter** [comment]
```
Nope, sorry, if the directory for writing is deleted, this is handled as a 
fatal error and the results are lost.  
```

Original comment by `niels.he...@gmail.com` on 24 Feb 2012 at 7:59
- Changed state: **WontFix**

**Comment by GoogleCodeExporter** [comment]
```
I lost report by same way/. :(
```

Original comment by `zaoro...@gmail.com` on 14 Feb 2014 at 11:57
